hue/hue/base/hue-mysql-pvc.yaml

```diff
@@ -0,0 +1,13 @@
+---
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: hue-mysql
+  annotations:
+    volume.beta.kubernetes.io/storage-class: "$(storage_class)"
```

Comment:
I am not sure if we need this. We had it for JupyterHub because we had one deployment (MOC) which did not set default storage class and then one (not sure, maybe early IDH?) where default was not what we wanted to use.

The problem with having storage class here like this is, that you will always need to provide it - since storage class can have 3 values - `null`, ``""`, actual name...so in this case if `storage_class` is not set, it would end up being `""` and fail deployment in most cases.

You can take a look at JupyterHub - we have an overlay `storageClass` which adds the definitions and parameter if enabled.

WDYT?

Reply:
We can put it into an overlay. I actually initially wanted to do that, however I kinda tried to keep the initial PR as compact as possible and not introduce unnecessary complexity. I know the storage class in data catalog is used in the Internal DH with different values for stage/prod, so I think this parameter still applies.

Reply:
Moved to overlay